### DISCUSSION TITLE
Killswitch

### DISCUSSION THREAD

**Discussion by ghost** [Eloston/ungoogled-chromium#2670] [General]
Hi there,

I have observed that all Chrome/Chromium executables have a killswitch that disables them a while (usually days) after a new version is released. The activated killswitch produces distortion of the screen display, at least on my system. Is there any intent to disable the killswitch? Is it even known?

Regards,
Klaus

**Comment by PF4Public** [comment]
Is it an issue with ungoogled-chromium or Chromium?

**Reply by ghost** [reply]
ungoogled-chromium, too.

**Reply by PF4Public** [reply]
You'll have to provide more details. I have never heard anything like that with ungoogled-chromium.

**Reply by tomasz1986** [reply]
> ungoogled-chromium, too.

Are you really sure about that? Ungoogled Chromium doesn't include any auto-update mechanism, so it shouldn't even know that a new version has been released.
